dist/amd-resolver.min.js

```diff
@@ -1,2 +1,9 @@
+!function(){"use strict";function e(e){this.settings=e||{}}var s=require("./file");e.prototype.resolve=function(e){var i,t,n,r,a,o,p,l=this.settings,m=l.shim||{},u=l.packages||[],f=l.paths&&l.paths[e]||e,h=e.split("!");for(e=h.pop(),r=e.replace(/[\/\\]+/g,"/").split("/"),a=r.shift(),o=r.join("/"),i=0,t=u.length;t>i;i++){if(n=u[i],n===a){f=a+"/main";break}if(n.name===a){f=n.location?n.location+"/":"",f+=a+"/"+(o||n.main||"main");break}}return m.hasOwnProperty(e)&&(p={name:m[e].exports||m[e].name||e,deps:m[e].imports||m[e].deps||[]}),{name:e,file:new s(s.addExtension(f,"js"),l.baseUrl),urlArgs:l.urlArgs,shim:p,plugins:h}},e.File=s,module.exports=e}();
+},{"./file":2}],2:[function(require,module,exports){
+!function(){"use strict";function t(r,o){var e,i;o=o||"",r=t.parseUri(r),r.protocol||!o?e=t.parseFileName(r.path):(o=t.parseUri(o),i=t.mergePaths(r.path,o?o.path:"/"),e=t.parseFileName(i)),this._file=r,this.protocol=r.protocol?r.protocol+r.protocolmark:void 0,this.name=e.name,this.path=e.path}t.prototype.toUrl=function(){var t=this;return(t.protocol||"")+(t.path||"")+t.name},t.parseUri=function(r){if(!r)throw new Error("Must provide a string to parse");return t.isHttpProtocol(r)?t.parseHttpProtocol(r):t.parseFileProtocol(r)},t.parseFileProtocol=function(r){var o=/^(?:(file:)(\/\/\/?))?(([A-Za-z-]+:)?[/\\d\w\.\s-]+)/gim.exec(r);return o.shift(),o[2]&&(o[2]=t.normalize(o[2])),{protocol:o[0],protocolmark:o[1],path:o[2],drive:o[3],href:r,uriParts:o}},t.parseHttpProtocol=function(r){var o=/^((https?:)(\/\/)([\d\w\.-]+)(?::(\d+))?)?([\/\\\w\.()-]*)?(?:([?][^#]*)?(#.*)?)*/gim.exec(r);return o.shift(),o[5]&&(o[5]=t.normalize(o[5])),{origin:o[0],protocol:o[1],protocolmark:o[2],hostname:o[3],port:o[4],path:o[5],search:o[6],hash:o[7],href:r,uriParts:o}},t.hasProtocol=function(t){return/^(?:(https?|file)(:\/\/\/?))/g.test(t)===!1},t.isFileProtocol=function(t){return/^file:/gim.test(t)},t.isHttpProtocol=function(t){return/^https?:/gim.test(t)},t.parseFileName=function(t){var r,o=t.replace(/([^/]+)$/gim,function(t){return r=t,""});return{name:r,path:o}},t.addExtension=function(r,o){var e=t.parseFileName(r),i=e.name.split(".");return 1===i.length&&o&&i.push(o),e.path+i.join(".")},t.replaceExtension=function(t,r){var o=/([^.\/\\]+\.)[^.]+$/;return t.match(o)?t.replace(o,"$1"+r):t+"."+r},t.normalize=function(t){for(var r=t.replace(/[\\/]+/g,"/").split("/"),o=r.length-1,e=0,i=[];o>=0;){if(o>0){if(".."===r[o]){o-=1,e++;continue}if("."===r[o]){o-=1;continue}}e&&(o-=e,e=0),i.unshift(r[o]),o--}return i.join("/")},t.mergePaths=function(r,o){return"/"===r[0]?t.normalize(r):(r=o&&r?o+"/"+r:r||o,t.normalize(r))},module.exports=t}();
+},{}]},{},[1])(1)
+});
```

Comment:
The reason this grew in lines, I think, is because the minification task we use now runs separately on each module and then concatenates them together, instead of running on the final output.